src/onnx/parse_pow.cpp

```diff
@@ -0,0 +1,69 @@
+#include <migraphx/onnx/op_parser.hpp>
+#include <migraphx/instruction.hpp>
+#include <migraphx/ranges.hpp>
+#include <migraphx/make_op.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+namespace onnx {
+
+auto compute_type(shape::type_t t1, shape::type_t t2)
+{
+    const static std::unordered_map<shape::type_t, int> op_order = {{shape::int8_type, 1},
```

Comment:
It looks `shape::type_t` cannot be used as a key since its not hashable. You should just use an int as the key and then convert the enum to an int with a `static_cast`.

Reply:
Thanks, just changed that.